data-import-from-s3/src/main/java/no/unit/nva/dataimport/ImportRequest.java

```diff
@@ -0,0 +1,63 @@
+package no.unit.nva.dataimport;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.JsonSerializable;
+
+public class ImportRequest implements JsonSerializable {
+
+    @JsonProperty("bucket")
+    private String bucket;
+    @JsonProperty("folderPath")
+    private String folderPath;
+    @JsonProperty("table")
+    private String table;
+
+    @JacocoGenerated
+    public ImportRequest() {
+
+    }
+
+    public ImportRequest(String bucket,
```

Comment:
@JsonCreator does not seem to work with AWS's serialiser for some reason. Added default serializer.